### PR TITLE
Primary Channel & Files folderAPI pages in V1.0 & beta

### PR DESCRIPTION
**Pull request by Y-Meenakshi**
Hi Nick, 
As suggested, I have included the namespace and created a new PR for the V1.0 API pages.
Also fixed the file naming in all lowercase as Laura suggested.

### CODE REVIEW

**Comment by Lauragra** [comment]
@Y-Meenakshi , please include the changelog, TOC, and parent resource updates. What is the merge status of this PR?

**Comment by Y-Meenakshi** [comment]
Hi @Lauragra,
I have added updated the TOC, reference pages with the links for the new API pages. I will also update the Changelog. It is in 'do not merge' status.

File: api-reference/v1.0/toc.yml [hunk 1]
```diff
@@ -4529,6 +4529,8 @@ items:
           href: api/user-list-joinedteams.md
         - name: List all teams
           href: /graph/teams-list-all-teams?context=graph/api/1.0
+        - name: Primary channel
+          href: api/team-primarychannel.md
```
Comment (nkramer) [inline api-reference/v1.0/toc.yml:4533]: team-**get**primarychannel.md

**Comment by Y-Meenakshi** [comment]
Moved changes to another PR

**Pull request closed without merging**